docs/_toc.yml

```diff
@@ -8,6 +8,7 @@ parts:
   - caption: Basics
     chapters:
     - file: 01_introduction/trailer
+    - file: 01_introduction/hardware
```

Comment:
I'm wondering if the document might fit better in an advanced section, e.g. in the GPU-acceleration section? We should certainly not introduce hardware-aspects before anything else.

Reply:
moved to a separate hardware section